qt-kit/merge.kit.d/qt6.yml

```diff
@@ -0,0 +1,53 @@
+# cat qt6.yml
+sources:
+# We need this to inherit the eclass used for kit cache JSON generation
+- name: "qt-kit"
```

Comment:
You need always the `core-kit` with eclasses to correctly generate the reposcan files and the elaborated ebuild with all elaborated variables.
You don't need instead the `qt-kit` as source.